controllers/search_controller.rb

```diff
@@ -13,7 +13,70 @@ class SearchController < ApplicationController
     end
 
     get '/metadata' do
```

Comment:
here you you need to harmonize with the existent `/search/ontologies` endpoint. 
<img width="1513" alt="image" src="https://github.com/user-attachments/assets/817f6081-bf81-434f-9f28-f61deeecacb9" />

Reply:
what do you mean by harmonize ? they are doing the same thing, so we can group them into one function

Reply:
yes group them into one function, or extract function that you call in both places, because they are not doing exactly the same thing